src/Microsoft.AspNet.Mvc.Xml/XmlDataContractSerializerOutputFormatter.cs

```diff
@@ -167,10 +169,7 @@ public override Task WriteResponseBodyAsync([NotNull] OutputFormatterContext con
             var tempWriterSettings = WriterSettings.Clone();
             tempWriterSettings.Encoding = context.SelectedEncoding;
 
-            var innerStream = context.HttpContext.Response.Body;
-
-            using (var outputStream = new NonDisposableStream(innerStream))
```

Comment:
The type `NonDisposableStream` is used in input formatters and so we cannot remove it entirely (while investigating this noticed that Xml input formatters use this but json does not...we should fix it)